DEVELOPMENT.md

```diff
@@ -6,10 +6,10 @@
 * Download and install [Android Studio] (https://developer.android.com/studio/index.html)
 * Set up [Android Development Environment] (https://spring.io/guides/gs/android/)
 * [Fork this repo] (https://help.github.com/articles/fork-a-repo/)
-* Copy GoIVCodeStyle.xml into the codestyles folder under the IntelliJ home directory (create if it does not exist)
- *  Windows: `\.\config\codestyles\`
- *  OS X: `~/Library/Preferences//codestyles/`
- *  Linux: `~/./config/codestyles/`
+* Copy GoIVCodeStyle.xml into the codestyles folder under the Android Studio config/codestyles directory *(create if it does not exist)*
+ *  Windows: `\%USERPROFILE%\.{ANDROID_STUDIO_FOLDER}\config\codestyles\`
+ *  OS X: `~/Library/Preferences/{ANDROID_STUDIO_FOLDER}/codestyles/`
+ *  Linux: `./{ANDROID_STUDIO_FOLDER}/config/codestyles/`
```

Comment:
Oops yeah, linux has a dot too, probably missed that out.
Like I've explained in the commit message, it could be AndroidStudioBeta, AndroidStudio1.5, AndroidStudio2.2Beta, AndroidStudio3.6, etc. As long as you update AndroidStudio, it'll be different.